charts/weave-gitops/crds/wego-system.yaml

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
This should no longer exist and be installed with Weave GitOps

Reply:
removed.